cluster/ci/config/prow/config.yaml

```diff
@@ -1012,10 +1012,10 @@ presubmits:
     agent: jenkins
     labels:
       master: ci.openshift.redhat.com
-    always_run: true
+    always_run: false
     context: ci/openshift-jenkins/integration
     rerun_command: "/test integration"
```

Comment:
Do you want this test to be triggered alongside the new?

Reply:
no, going to move to "old-integration" and then remove when it's truly gone.